Project3/Part_A/Heap.h

```diff
@@ -0,0 +1,45 @@
+#include <vector>
+
+template <typename T>
+class heap {
+public:
+  void initializeMaxHeap(void) { buildMaxHeap(); }
+  T getItem(int i) { return item_list[i]; }
+  void insert(T value) { item_list.push_back(value); }
+  int size(void) { return item_list.size(); }
+  void heapsort(void) 
+  {
+    buildMaxHeap();
+    int size = item_list.size();
+    for (int i = size; i > 1; i--) {
+      std::swap(item_list[1], item_list[i]);
+      size -= 1;
+      maxHeapify(1, size);
+    }
+  }
```

Comment:
method definitions should be in a separate file (see [style guidelines](https://northeastern.instructure.com/courses/153057/files/23232078?module_item_id=9502079))